version1/04 - Web services.md

```diff
@@ -364,6 +364,12 @@ Content-Type: application/json
         - field ID placeholder which identifies a business-specific metadata field
         - **values**
           - array of zero or more values to write to the specified business-specific metadata field
+  - we have screening inheritance logic build in the service(create only) for Document, Record and File types, the future object screening is determined by rules of priority:
```

Comment:
Types should be specified in Norwegian

Reply:
> Types should be specified in Norwegian

https://github.com/documaster/noark5-web-services/pull/59/commits/63f0165a2e8d089a2626a137635b09f3a07f69cd